sdk/storage/Azure.Storage.Files.DataLake/samples/Sample01a_HelloWorld.cs

```diff
@@ -0,0 +1,428 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using Azure.Storage;
+using Azure.Storage.Files.DataLake;
+using Azure.Storage.Files.DataLake.Models;
+using NUnit.Framework;
+
+namespace Azure.Storage.Files.DataLake.Samples
+{
+    /// <summary>
+    /// Basic Azure DataLake Storage samples.
+    /// </summary>
+    public class Sample01a_HelloWorld : SampleTest
+    {
+        /// <summary>
+        /// Upload a file to a DataLake File.
+        /// </summary>
+        [Test]
+        public void Append()
+        {
+            // Create three temporary Lorem Ipsum files on disk that we can upload
+            int oneThirdPosition = SampleFileContent.Length / 3;
+            string sampleFileContentPart1 = CreateTempFile(SampleFileContent.Substring(0, oneThirdPosition));
+            string sampleFileContentPart2 = CreateTempFile(SampleFileContent.Substring(oneThirdPosition, oneThirdPosition + 1));
+            string sampleFileContentPart3 = CreateTempFile(SampleFileContent.Substring((oneThirdPosition * 2 + 1), oneThirdPosition + 1));
+
+            // Make StorageSharedKeyCredential to pass to the serviceClient
+            string storageAccountName = StorageAccountName;
+            string storageAccountKey = StorageAccountKey;
+            Uri serviceUri = StorageAccountBlobUri;
+
+            StorageSharedKeyCredential sharedKeyCredential = new StorageSharedKeyCredential(storageAccountName, storageAccountKey);
+
+            // Get a reference to a FileSystemClient
+            DataLakeServiceClient serviceClient = new DataLakeServiceClient(serviceUri, sharedKeyCredential);
+
+            // Get a reference to a filesystem named "sample-filesystem-append" and then create it
+            DataLakeFileSystemClient filesystem = serviceClient.GetFileSystemClient("sample-filesystem-append");
+            filesystem.Create();
+            try
+            {
+                // Get a reference to a file named "sample-file" in a filesystem named "sample-filesystem"
+                DataLakeFileClient file = filesystem.GetFileClient(Randomize("sample-file"));
+
+                // Open the file and upload its data
+                file.Create();
+
+                file.Append(File.OpenRead(sampleFileContentPart1), 0);
+                file.Append(File.OpenRead(sampleFileContentPart2), oneThirdPosition);
+                file.Append(File.OpenRead(sampleFileContentPart3), oneThirdPosition * 2 + 1);
+                file.Flush(SampleFileContent.Length);
+
+                // Verify we uploaded one file with some content
+                Assert.AreEqual(1, filesystem.ListPaths().Count());
```

Comment:
Let's move this ListPaths() call above the append.  The file is actually created with the file create.